server/src/main/java/io/unitycatalog/server/persist/SchemaRepository.java

```diff
@@ -197,7 +197,9 @@ public SchemaInfo updateSchema(String fullName, UpdateSchema updateSchema) {
                 ErrorCode.ALREADY_EXISTS, "Schema already exists: " + updateSchema.getNewName());
           }
         }
-        if (updateSchema.getComment() == null && updateSchema.getNewName() == null) {
+        if (updateSchema.getComment() == null
+            && updateSchema.getNewName() == null
+            && (updateSchema.getProperties() == null || updateSchema.getProperties().isEmpty())) {
           tx.rollback();
           return convertFromDAO(schemaInfo, fullName);
```

Comment:
We should rename `schemaInfo` to `schemaInfoDAO` at line [190](https://github.com/unitycatalog/unitycatalog/pull/190/files#diff-3220d6f332348a34e53ba25cec22a52726afa71cf8f7d4fc31c55b1aee34477bR190) to avoid any confusion and correspondingly update all it's occurrences as follows:
```suggestion
          return convertFromDAO(schemaInfoDAO, fullName);
```